connector/MongoDBAtlasODBC.query-test.query.pq

```diff
@@ -1,329 +0,0 @@
-section MongoDBAtlasODBCQueryTests;
```

Comment:
Since we now support native query as opposed to the previous query function, how about we modify the test to confirm the results from the native query are correct.  
Have you tried running this test with:
```
= Value.NativeQuery( Datasource_Database, "SELECT * FROM sales")
```

Reply:
I can do that; great suggestion. It was an oversight on my part.